### DISCUSSION TITLE
Pacstall

### DISCUSSION THREAD

**Discussion by trymeouteh** [AntiMicroX/antimicrox#430] [Polls]
Please add this app into the pacstall repository. Pacstall is the AUR for Ubuntu which will allow Ubuntu users to easily install the app and receive updates very shortly after an update is released.

https://pacstall.dev/

**Comment by pktiuk** [comment]
Flatpak is for Ubuntu users willing to use the latest version.  
It may be a good idea, but I don't have time for dealing with another package distribution system.  

But if anyone wants to add it, I won't oppose.
